webboot/webboot.go

```diff
@@ -178,6 +178,19 @@ func usage() {
 func main() {
 	flag.Parse()
 
+	commandline, err := ioutil.ReadFile("/proc/cmdline")
+
+
+	if err != nil {
+		fmt.Errorf("Unable to read /proc/cmdline: %v", err)
+		//maybe here, we can have a hardcoded commandline so if
```

Comment:
nope. If we can't read /proc/cmdline we are done here.

Reply:
and now that you've see how this works, a slight set of changes. First, change this as follows
add an import
"github.com/u-root/u-root/pkg/cmdline"

Change this code to this:
		cl := cmdline.NewCmdLine()
		if cl.Err != nil {
			log.Fatal(cl.err)
		}

There is code at line 248:
change lines 248-252 as follows (yes, those 5 lines become 1 line)

			bookmark[filename].Cmdline = cl + " " + bookmark[filename].Cmdline

Later, we're going to fix the bookmarks to have minimal text, but this will do for now.

Reply:
i don't know what you mean. you cannot take `cl` which is `cmdline.Cmdline` and put it into bookmark as a string

Reply:
yeah, sorry.
bookmark[filename].Cmdline = cl.Cmdline + " " + bookmark[filename].Cmdline